### ISSUE TITLE
Install apps in order of selection

### ISSUE THREAD

**Issue by alessiopellindgm** [EOS-Solutions/Gordon#59]
**Is your feature request related to a problem? Please describe.**
In smeagol, installing or updating apps from the feed followed the app selection order and not the list view order. This is useful when you need to update many apps that are dependent using the sequence indicated by DevOps.

**Describe the solution you'd like**
In image example, I selected the apps in this order:
1. Common data layer
2. Doc shipping Common Data Layer
3. Commissions
5. Commissions for Italy

**Product**
- [X] Gordon WebApp
- [ ] Gordon Powershell
- [ ] Bare.WebApi

**Additional context**
![image](https://github.com/EOS-Solutions/Gordon/assets/97117499/7dc95291-acd9-4d99-bc82-e92a85d65011)

**Comment by Stefano-Primo-EOS** [comment]
Gordon sends the app to the "Core" with the selected order. 
Then the core, sorts and adds needed apps for the dependency resolution. 

So, in your case, you can directly choose to install only the "Commission for Italy", then gordon will check: Which app is needed for this app? and it will build the dependency tree as needed

**Comment by alessiopellindgm** [comment]
Okay. For updating already published apps, it's the best solution and have all dependent apps correctly aligned.
In development it's complicated since I don't have a released version and I can't link the dependency version correctly.

Does updating the calculated apps in the dependency tree update to the latest version app of the feed or by evaluating the minimun dependency and skip latest version?
I do test and try to give you feedback.

**Comment by alessiopellindgm** [comment]
Update (use "Minimum Version" and "Latest Version" ) return an error:

Log
```
Getting apps from Service
Retrived metadata from service (19651 ms)
Get app from service finished (2 ms)
Getting apps from Feed
Retrived metadata from feed (6 ms)
Get app from feed finished (0 ms)
Getting apps from Service
Retrived metadata from service (21335 ms)
Get app from service finished (11 ms)
Getting apps from Feed
Retrived metadata from feed (304 ms)
Get app from feed finished (1 ms)
Resolving dependencies using policy 'MinimumVersion'
This operation cannot be performed on a default instance of ImmutableArray<T>. Consider initializing the array, or checking the ImmutableArray<T>.IsDefault property.
Getting apps from Service
Retrived metadata from service (16266 ms)
Get app from service finished (0 ms)
Getting apps from Feed
Retrived metadata from feed (40 ms)
Get app from feed finished (0 ms)
Resolving dependencies using policy 'LatestVersion'
This operation cannot be performed on a default instance of ImmutableArray<T>. Consider initializing the array, or checking the ImmutableArray<T>.IsDefault property.
Getting apps from Service
```

**Comment by Stefano-Primo-EOS** [comment]
Fix in 1.0.143

**Comment by alessiopellindgm** [comment]
> Fix in 1.0.143

Confirm.
